packages/ember-htmlbars/lib/component.js

```diff
@@ -138,6 +139,21 @@ const Component = View.extend(TargetActionSupport, {
     }
   }),
 
+  templateForName(name, type) {
```

Comment:
This is still used for component registration and is exercised in this [test](https://github.com/emberjs/ember.js/blob/master/packages/ember/tests/component_registration_test.js#L154-L172).